src/lightkurve/interact.py

```diff
@@ -119,12 +119,15 @@ def prepare_tpf_datasource(tpf, aperture_mask):
         Bokeh object to be shown.
     """
     npix = tpf.flux[0, :, :].size
-    pixel_index_array = np.arange(0, npix, 1).reshape(tpf.flux[0].shape)
-    xx = tpf.column + np.arange(tpf.shape[2])
-    yy = tpf.row + np.arange(tpf.shape[1])
+    ncadences, ny, nx = tpf.shape
+    pixel_index_array = np.arange(0, npix, 1)
+    xx = tpf.column + np.arange(nx)
+    yy = tpf.row + np.arange(ny)
     xa, ya = np.meshgrid(xx, yy)
+    xa = xa.flatten()
```

Comment:
Maybe a 1 line comment would be helpful, explaining why `xa`, `ya` and `tpf_source.selected.indices` are flattened here, when someone read the codes say a year from now.